Sources/SourceControl/RepositoryManager.swift

```diff
@@ -97,8 +97,7 @@ public class RepositoryManager: Cancellable {
         repository: RepositorySpecifier,
         updateStrategy: RepositoryUpdateStrategy,
         observabilityScope: ObservabilityScope,
-        delegateQueue: DispatchQueue,
-        callbackQueue: DispatchQueue
+        delegateQueue: DispatchQueue
```

Comment:
https://github.com/swiftlang/swift-package-manager/pull/8721 should refactor this so `lookup` is all async.